Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -170,8 +170,24 @@ export class ResourceGroup {
             tl.error(error.message);
             if (error.details) {
                 tl.error(tl.loc("Details"));
+                var isPolicyVoilated = false;
                 for (var i = 0; i < error.details.length; i++) {
-                    var errorMessage = util.format("%s: %s %s", error.details[i].code, error.details[i].message, error.details[i].details);
+                    var errorMessage = null;
+                    if (error.details[i].code === "RequestDisallowedByPolicy") {
+                        if (!isPolicyVoilated) {
```

Comment:
Is this logic to print the first policy violation? You can break here without this variable as well, right?

Reply:
This logic is used to print the URL to policy violated if the error "RequestDisallowedByPolicy" appears even once. And we cannot break here because we have to modify all the policy related errors.

Reply:
If that's the case, shouldn't `isPolicyVoilated = true;` go inside the if block?

Reply:
Why not print all the policy links